### ISSUE TITLE
Some questions on example/distributed-training results

### ISSUE THREAD

**Issue by cuihenggang** [dmlc/mxnet#595]
Hi,

I have some questions for the results in example/distributed-training.

For the section (table) of "5 machines with 10 GTX 980, BSP", it uses "batch_size=96", but for the "System Performance" table, it uses "batch_size=48" for the same system setting (10 GTX980 GPUs). I'm wondering does it mean we can fit a batch_size of 96 in GPU memory, then why don't we use batch_size 96 for the "System Performance" result also?

And I assume the "time" column in the "5 machines with 10 GTX 980, BSP" table means per-epoch time?

Also, to get the training and validation results, do we just set the initial learning rate and let it run? I'm asking this because I remember in another post about training ImageNet on a single GPU, we need to tune the learning rate during the run, so I'm wondering is it also this case for distributed training.

Nice results.

Thanks,
Cui

**Comment by mli** [comment]
for the performance, it is "batch size / GPU" namely the batch size given to a single gpu. the number 48 is consistent with 96 shown later, because each machine has two gpus. 

the time is per epoch. i should remove the time column later

i simply fix the learning rate for the first 20 iterations, and then change it later.

**Comment by cuihenggang** [comment]
That makes sense.

PS. Caffe should also support the "Batch Normalization" specialization. It improves the accuracy so much.

Thank you!
Cui

**Comment by mli** [comment]
I'm going to close the issue now. Please feel free to reopen it if you have other questions.
